src/ui/Assets/Javascript/character-count.js

```diff
@@ -0,0 +1,328 @@
+      // var tokens = text.split(' ')
+      // length = tokens.length-1
+      var tokens = text.match(/\S+/g) || [] // Matches consecutive non-whitespace chars
+      length = tokens.length
+    } else {
+      length = text.length
+    }
+    return length
+  }
+
+  // Highlight text from a specific limit to end
+  CharCount.prototype.highlight = function (text, limit) {
+    text = text.replace(/\n$/g, '\n\n')
+    var textBeforeLimit = text.slice(0, limit).escape()
+    var textAfterLimit = text.slice(limit).escape()
+    text = [textBeforeLimit, '<mark>', textAfterLimit, '</mark>'].join('')
+    return text
+  }
+
+  // Generate count message and bind it to the input
+  // returns reference to the generated element
+  CharCount.prototype.createCountMessage = function (countElement) {
+    var elementId = countElement.getAttribute('id')
+    // Check for existing info count message
+    var countMessage = document.getElementById(elementId + '-info')
+    // If there is no existing info count message we add one right after the field
+    if (elementId && !countMessage) {
+      countElement.insertAdjacentHTML('afterend', '<span id="' + elementId + '-info" class="govuk-hint govuk-c-charcount__message" aria-live="polite"></span>')
+      countElement.setAttribute('aria-describedby', elementId + '-info')
+      countMessage = document.getElementById(elementId + '-info')
+    }
+    return countMessage
+  }
+
+  // Bind input propertychange to the elements and update based on the change
+  CharCount.prototype.bindChangeEvents = function (countElementExtended) {
+    if (countElementExtended.countElement.addEventListener) {
+      // W3C event model
+      // countElementExtended.countElement.addEventListener('input', CharCount.prototype.handleInput.bind(countElementExtended))
+      // countElementExtended.countElement.addEventListener('onpropertychange', CharCount.prototype.updateCountMessage.bind(countElementExtended))
+      // IE 9 does not fire an input event when the user deletes characters from an input (e.g. by pressing Backspace or Delete, or using the "Cut" operation).
+      countElementExtended.countElement.addEventListener('keyup', CharCount.prototype.updateCountMessage.bind(countElementExtended))
+    } else {
+      // Microsoft event model: onpropertychange/onkeyup
+      countElementExtended.countElement.attachEvent('onkeyup', CharCount.prototype.handleInput.bind(countElementExtended))
+    }
+
+    // Bind scroll event if highlight is set
+    if (countElementExtended.options.highlight === true) {
+      countElementExtended.countElement.addEventListener('scroll', CharCount.prototype.handleScroll.bind(countElementExtended))
+      window.addEventListener('resize', CharCount.prototype.handleResize.bind(countElementExtended))
+    }
+
+    // Bind focus/blur events for polling
+
+    // $(countElementExtended.countElement).blur(function() {
+    //   CharCount.prototype.updateCountMessage(countElementExtended);
+    // });
```

Comment:
(You could probably delete my hack to get it working with Simple MDE in the prototype.)